server/src/mindustry/server/ServerControl.java

```diff
@@ -61,7 +61,7 @@ public ServerControl(String[] args){
             "bans", "",
             "admins", "",
             "shufflemode", "custom",
-            "globalrules", "{reactorExplosions: false}"
+            "globalrules", "{reactorExplosions: false}" //schematicAllowed: true} //is not working
```

Comment:
It's true by default, why add it here in the first place?

Reply:
I removed the comment but I couldn't find a way to add an extra argument.